mobile/src/home-view.js

```diff
@@ -98,8 +98,7 @@ class HomeView extends Component {
               sessions[i].key = data.key
               if (this.state.session.key === data.key) {
                 let changeViewStatus = false
-                var newSession = this.state.session
-                newSession = data.val()
+                let newSession = data.val()
                 newSession.key = data.key
                 if (this.state.session.archive === false && data.val().archive) {
                   changeViewStatus = true
```

Comment:
Line 105 has `newSession = ""`.  Looks like it is going from being an Object to being a string?

Reply:
I store blank objects in many cases as blank strings. Would {} operate the same way as ""?

Reply:
No.  `{}` is truthy.  `""` is falsy.  `null` is falsy.  If you want to store the fact that something that is normally an Object has no value, `null` is preferable (then you check `== null` anywhere that you expect that it might be)